evennia/commands/default/building.py

```diff
@@ -868,7 +871,8 @@ def func(self):
         "Implements the tunnel command"
 
         if not self.args or not self.lhs:
-            string = "Usage: @tunnel[/switch] <direction> [= roomname[;alias;alias;...][:typeclass]]"
+            string = "Usage: @tunnel[/switch] <direction> [= <roomname>[" \
+                     ";alias;alias;...][:typeclass]]"
```

Comment:
Same here, I think the lone `[` belongs on the next line in the case of usage strings. This goes for all cases where this happens.